app/build.gradle

```diff
@@ -25,6 +25,9 @@ android {
         versionCode versions.code // Notice that you may want to use BUILD_NUMBER from CI in real project with own CI.
         versionName versions.name
 
+        // Make generated apks contain version name and code
+        setProperty("archivesBaseName", "-${versions.name}-${versions.code}")
```

Comment:
Is there a way to do this without `setProperty()`? Like just `archivesBaseName = "-${versions.name}-${versions.code}"`?